src/views/InputView.js

```diff
@@ -0,0 +1,7 @@
+import { Console } from '@woowacourse/mission-utils';
+
+export class InputView {
+  static readInput(MESSAGE) {
```

Comment:
```suggestion
  static readInput(message) {
```
인자는 런타임 중에 바뀌고 어떤 값이 올지 모르는데 상수 컨벤션으로 적혀있는게 약간 어색한 것 같아요!